src/commands/ptal.ts

```diff
@@ -80,22 +102,68 @@ const generateReplyFromInteraction = async (description: string, github: string,
 				.setURL(githubURL.href);
 
 			components.push(githubLink);
+			
+			const [owner, repo, id] = pathSections;
+			const pull_number = Number.parseInt(id);
+			const pr_info = {
+				owner,
+				repo,
+				pull_number
+			}
 
 			try
 			{
-				let pr = await octokit.rest.pulls.get({owner: pathSections[0], repo: pathSections[1], pull_number: Number.parseInt(pathSections[3])});
+				let pr = await octokit.rest.pulls.get(pr_info);
 				embed.setTitle(pr.data.title);
 
-				let files = (await octokit.rest.pulls.listFiles({owner: pathSections[0], repo: pathSections[1], pull_number: Number.parseInt(pathSections[3])})).data;
+				let pr_state: PullRequestState = 'PENDING';
+				let reviews: string[] = [];
+				if (pr.data.state === 'closed') {
+					if (pr.data.merged)
+					{
+						pr_state = 'MERGED';
+					}
+					else
+					{
+						pr_state = 'CLOSED';
+					}
+				}
+				else
+				{
+					if (pr.data.review_comments > 0)
+					{
+						let { data: reviews } = await octokit.rest.pulls.listReviews(pr_info);
+						for (let { state: reviewState } of reviews) {
```

Comment:
Could you explain the syntax used here? I've never seen it before and it seemingly overwrites the string[] type of reviews, resulting on errors on line 138 and 146